src/containers/App.js

```diff
@@ -49,7 +49,7 @@ class App extends Component {
 }
 
 
-export default connect(state => ({
+export default withRouter(connect(state => ({
```

Comment:
Could you explain for what do you use extra `withRouter` method here?

Reply:
withRouter adds history props to a wrapped component.
I fixed the root of the problem. Thanks for review